modules/solid_mechanics/doc/content/source/constraints/NodalFrictionalConstraint.md

```diff
@@ -3,7 +3,7 @@
 !syntax description /Constraints/NodalFrictionalConstraint
 
 Only a penalty formulation is implemented.
-First the previous time step tangeantial force is computed as:
+First. the previous time step tangential force is computed as:
```

Comment:
My original suggestion must have had a typo. I think this should be a comma, not a period after the word `First`